plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/OpenShiftCommonImages.java

```diff
@@ -39,5 +39,11 @@
 	public static final ImageDescriptor SYSTEM_PROCESS = repo.create("systemprocess.gif"); //$NON-NLS-1$ 
 	public static final Image SYSTEM_PROCESS_IMG = repo.getImage("systemprocess.gif"); //$NON-NLS-1$ 
 
+	public static final ImageDescriptor WARNING = repo.create("pficon-warning.png"); //$NON-NLS-1$ 
```

Comment:
@adietish these are the icons used in the 'summary' after the resources are created.  I would be happy to use the eclipse standard if I could figure out what those were.  It's not obvious to me 1) where to find the icons.  2) what represents warning and success in this context.  Is there a style guide that can provide this info? https://wiki.eclipse.org/User_Interface_Guidelines ???